carbon-design-system/carbon
# feat(tile): add light prop

Closes #4122 

Add light prop to Tile component for use on `$ui-01` container.

#### Changelog

**New**

- `light` prop in all Tile components (`Tile`, `ClickableTile`, `SelectableTile`, `RadioTile`, `ExpandableTile`)

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

#### Testing / Reviewing

- Pull PR
- Make sure packages are built (`yarn build`)
- Run react storybook
```sh
cd packages/react
yarn storybook
```
- Activate light prop in knobs for Tile -> Default, Clickable, Multi-select, Selectable and Expandable. The tile background should change to the `$ui-02` equivalent color of the selected carbon theme.
